datasafe-business/src/test/java/de/adorsys/datasafe/business/impl/e2e/SchemeDelegationWithDbTest.java

```diff
@@ -101,13 +103,15 @@ void testProfileOnDbDataOnFsWorks() {
                 "users/john/private/keystore",
                 "users/john/private/files",
                 "users/john/private/files/SIV",
-                fsPath.relativize(encryptedFile).toString()
+                fsPath.relativize(encryptedFile).toString().replaceAll("\\\\","/")
             );
     }
 
     private Stream<String> listDb(String path) {
-        return db.list(BasePrivateResource.forAbsolutePrivate(URI.create(path)))
-            .map(it -> it.location().asURI().toString());
+        try (Stream<AbsoluteLocation<ResolvedResource>> stream = db.list(BasePrivateResource.forAbsolutePrivate(URI.create(path)))){
```

Comment:
Try-with-resources is meaningless here - opened stream is manipulated elsewhere, better to return List from this method and keep try-with-resources